packages/bundle/package-lock.json

```diff
@@ -3624,12 +3624,14 @@
 				"balanced-match": {
 					"version": "1.0.0",
 					"bundled": true,
-					"dev": true
+					"dev": true,
+					"optional": true
```

Comment:
`optional` is expected to be in `package-lock.json`, but a bug in NPM platform make them gone in previous commits.